src/requestforattestation/RequestForAttestation.ts

```diff
@@ -90,17 +91,17 @@ export default class RequestForAttestation implements IRequestForAttestation {
     this.claimerSignature = this.sign(identity)
   }
 
-  public removeClaimProperties(properties: string[]) {
+  public removeClaimProperties(properties: string[]): void {
     properties.forEach(key => {
-      if (!this.claimHashTree.hasOwnProperty(key)) {
+      if (!this.claimHashTree[key]) {
```

Comment:
Not sure whether that can happen, but what if `this.claimHashTree[key]` exists and is `0` or `false`?

Reply:
this can't happen. But maybe the claimHashTree should be better defined, type-wise

Reply:
https://github.com/KILTprotocol/sdk-js/issues/152